src/ipc/StartListening.cc

```diff
@@ -39,7 +39,8 @@ Ipc::StartListening(int sock_type, int proto, const Comm::ConnectionPointer &lis
     Must(cbd);
     cbd->conn = listenConn;
 
-    if (UsingSmp()) { // if SMP is on, share
+    if ((listenConn->flags & COMM_REUSEPORT) && UsingSmp()) {
```

Comment:
After this change (done in accordance with the specs) port sharing is disabled by default (http_port without new worker-queues option). Is this correct?